src/main/java/rocks/turncodr/mycurriculum/model/ModuleType.java

```diff
@@ -1,13 +1,16 @@
 package rocks.turncodr.mycurriculum.model;
 
+/**
+ *  Attribute for Module.
+ */
 public enum ModuleType {
     MAIN_MODULE(0),
     PLACEHOLDER_MODULE(1),
     ELECTIVE_MODULE(2);
 
     private int index;
```

Comment:
Why do you need this index? Enums have an ordinal number starting at 0 for the first Object. Why do you not use that?